src/Microsoft.ServiceFabric.Services.Remoting/V2/ServiceRemotingMessageSerializersManager.cs

```diff
@@ -1,94 +1,94 @@
-            {
-                headerSerializer = new ServiceRemotingMessageHeaderSerializer(new BufferPoolManager());
-            }
-
-            if (serializationProvider == null)
-            {
-                if (useWrappedMessage)
-                {
-                    serializationProvider = new WrappingServiceRemotingDataContractSerializationProvider();
-                }
-                else
-                {
-                    serializationProvider = new ServiceRemotingDataContractSerializationProvider();
-                }
-            }
-
-            this.serializationProvider = serializationProvider;
-            this.cachedBodySerializers = new ConcurrentDictionary<int, CacheEntry>();
-            this.headerSerializer = headerSerializer;
-        }
-
-        public IServiceRemotingMessageSerializationProvider GetSerializationProvider()
-        {
-            return this.serializationProvider;
-        }
-
-        public IServiceRemotingMessageHeaderSerializer GetHeaderSerializer()
-        {
-            return this.headerSerializer;
-        }
-
-        public IServiceRemotingRequestMessageBodySerializer GetRequestBodySerializer(int interfaceId)
-        {
-            return this.cachedBodySerializers.GetOrAdd(interfaceId, this.CreateSerializers).RequestBodySerializer;
-        }
-
-        public IServiceRemotingResponseMessageBodySerializer GetResponseBodySerializer(int interfaceId)
-        {
-            return this.cachedBodySerializers.GetOrAdd(interfaceId, this.CreateSerializers).ResponseBodySerializer;
-        }
-
-        internal virtual CacheEntry CreateSerializers(int interfaceId)
-        {
-            var interfaceDetails = this.GetInterfaceDetails(interfaceId);
-
-            // get the service interface type from the code gen layer
-            var serviceInterfaceType = interfaceDetails.ServiceInterfaceType;
-
-            // get the known types from the codegen layer
-            var requestBodyTypes = interfaceDetails.RequestKnownTypes;
-
-            // get the known types from the codegen layer
-            var responseBodyTypes = interfaceDetails.ResponseKnownTypes;
-
-            return new CacheEntry(
-                this.serializationProvider.CreateRequestMessageSerializer(serviceInterfaceType, requestBodyTypes, interfaceDetails.RequestWrappedKnownTypes),
-                this.serializationProvider.CreateResponseMessageSerializer(serviceInterfaceType, responseBodyTypes, interfaceDetails.ResponseWrappedKnownTypes));
-        }
-
-        internal virtual InterfaceDetails GetInterfaceDetails(int interfaceId)
-        {
-            if (!ServiceCodeBuilder.TryGetKnownTypes(interfaceId, out var interfaceDetails))
-            {
-                throw new ArgumentException("No interface found with this Id  " + interfaceId);
-            }
-
-            return interfaceDetails;
-        }
-    }
-}
+// ------------------------------------------------------------

```

Comment:
Could you double-check that we're not reverting from LF to CRLF line endings in the Git index?